docs/usage/model_config.md

```diff
@@ -263,7 +263,7 @@ print(Character.model_json_schema(by_alias=True))
 {
     'type': 'object',
     'properties': {
-        'ActorName': {'type': 'string', 'default': None, 'title': 'Actorname'},
+        'Name': {'type': 'string', 'default': None, 'title': 'Name'},
```

Comment:
I don't think this attribute name should be changed, because the alias generator isn't supposed to override explicitly-set aliases.

The relevant attribute is `FieldInfo.alias_priority`.

I think this shouldn't be too hard to fix, but we should look at how that is used now.

Reply:
I think we probably want to make a version of `pydantic._internal._model_construction.apply_alias_generator` for dataclasses / typeddict